homeassistant/components/vartastorage/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""The VARTA Storage integration."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+from vartastorage.vartastorage import VartaStorage
+
+from homeassistant import config_entries, core
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+
+PLATFORMS: list[str] = ["sensor"]
```

Comment:
Use constants for platforms definition:

```suggestion
from homeassistant.const import Platform

PLATFORMS: list[Platform] = [Platform.SENSOR]
```

Reply:
done

Reply:
Are you sure you pushed your changes? @Vip0r. I wanted to give it another review, however it seems that not all current comments (that are dismissed) are addressed.